strongback/src/org/strongback/command/Command.java

```diff
@@ -149,6 +150,34 @@ final boolean isInterruptible() {
     }
 
     /**
+     * Create a command that uses the supplied PID controller and moves within the specified tolerance of the specified
+     * setpoint.
+     *
+     * @param controller the PID+FF controller; may not be null
+     * @param setpoint the desired value for the input to the controller
+     * @param tolerance the absolute tolerance for how close the controller should come before completing the command
+     * @return the command; never null
+     */
+    public static Command approach(PIDController controller, double setpoint, double tolerance) {
```

Comment:
I like `control`. 

`runController` indicates purpose very well if you like that option.